### PR TITLE
[IMP] add code button 07xxx

### PR DESCRIPTION
**Pull request by houssine78**
https://gestion.coopiteasy.be/web#id=4896&view_type=form&model=project.task

### CODE REVIEW

File: spp_custom/__openerp__.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 {
     "name": "SPP Customizations",
-    "version": "9.0.1.0.0",
+    "version": "9.0.1.0.1",
     "depends": ["beesdoo_base", "beesdoo_product"],
     "author": "Coop IT Easy SCRLfs",
     "license": "AGPL-3",
     "category": "",
-    "website": "www.coopiteasy.be",
+    "website": "https://www.coopiteasy.be",
```
Comment (robinkeunen) [inline spp_custom/__openerp__.py:12]: nazi

File: spp_custom/models/product.py [hunk 2]
```diff
@@ -65,3 +65,10 @@ def generate_ref_code_non_bio(self):
             "spp_custom.seq_ean_product_internal_ref_weight_non_bio"
         )
         self.generate_ref_code("09", sequence)
+
+    @api.multi
+    def generate_ref_code_non_food(self):
+        sequence = self.env.ref(
+            "spp_custom.seq_ean_product_internal_ref_weight_non_food"
+        )
+        self.generate_ref_code("07", sequence)
```
Comment (robinkeunen) [inline spp_custom/models/product.py]: missing eol

**Review by robinkeunen — APPROVED** [review]
missing eol, otherwise 👍

File: spp_custom/models/product.py (hunk 2 above)
Comment (houssine78) [reply spp_custom/models/product.py]: done

**Pull request merged**